rocAL/include/meta_data/meta_data_reader.h

```diff
@@ -80,6 +82,10 @@ struct MetaDataConfig
     unsigned out_img_height() const { return _out_img_height; }
     void set_out_img_width(unsigned out_img_width) { _out_img_width = out_img_width; }
     void set_out_img_height(unsigned out_img_height) { _out_img_height = out_img_height; }
+    std::vector<std::string> file_names() const { return _file_names; }
+    std::vector<int> labels() const { return _labels; }
+    void set_file_names(std::vector<std::string> file_names) { _file_names = file_names; }
```

Comment:
Please take these arguments by reference instead since it is passed by reference

Reply:
Done